src/rust/engine/fs/src/directory.rs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the Apache License, Version 2.0 (see LICENSE).
 
 use std::collections::HashMap;
+use std::convert::TryFrom;
```

Comment:
This should already be in the prelude for the Rust 2021 edition.

Reply:
It's imported a few places: I'll strip in a separate change.